jib-core/src/test/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilderTest.java

```diff
@@ -84,12 +85,43 @@ private static void verifyNextTarArchiveEntryIsDirectory(
     assertThat(extractionPathEntry.getMode()).isEqualTo(TarArchiveEntry.DEFAULT_DIR_MODE);
   }
 
-  private static FileEntry defaultLayerEntry(Path source, AbsoluteUnixPath destination) {
+  /**
+   * Verifies that the modification time has been reset in the PAX headers.
+   *
+   * @param entry The archive entry in the layer.
+   */
+  private static void verifyThatModificationTimeIsReset(ArchiveEntry entry) {
+    assertThat(entry.getLastModifiedDate().toInstant())
+        .isEqualTo(FileEntriesLayer.DEFAULT_MODIFICATION_TIME);
+  }
+
+  private static FileEntry layerEntry(
+      Path source, AbsoluteUnixPath destination, FilePermissions permissions, String ownership)
+      throws IOException {
     return new FileEntry(
         source,
         destination,
-        FileEntriesLayer.DEFAULT_FILE_PERMISSIONS_PROVIDER.get(source, destination),
-        FileEntriesLayer.DEFAULT_MODIFICATION_TIME);
+        permissions,
+        // Here we make sure to use a default modification time
+        // since JIB will set modification time for all files to
+        // the one set in JIB settings by a user. In case user
+        // hasn't set it, JIB will use default modification time,
+        // i.e. 1 second past January 1st, 1970 (1 second past epoch).
+        FileEntriesLayer.DEFAULT_MODIFICATION_TIME,
+        ownership);
+  }
+
+  private static FileEntry layerEntry(
+      Path source, AbsoluteUnixPath destination, FilePermissions permissions) throws IOException {
+    return layerEntry(source, destination, permissions, "");
+  }
+
+  private static FileEntry layerEntry(Path source, AbsoluteUnixPath destination)
+      throws IOException {
+    return layerEntry(
+        source,
+        destination,
+        FileEntriesLayer.DEFAULT_FILE_PERMISSIONS_PROVIDER.get(source, destination));
```

Comment:
Just revert this file back to its original state. The original author misunderstood this part. The PR shouldn't touch this file, and the existing tests will verify the correctness of your changes.